src/auto_add_domain.py

```diff
@@ -0,0 +1,48 @@
+import json
```

Comment:
since this is an ad hoc function (used only for this annotation), I would prefer if we don't check this in.

Reply:
I added it because it seems that it could be useful for other people working on #4 who would likely need it.